File: src/help_requests/number_of_good_pairs/DummySolution.java
package help_requests.number_of_good_pairs;

public class DummySolution {
    public static long numberOfGoodPairs(int[] x, int[] y) {
        int n = x.length;
        long ans = 0;
        for (int i = 0; i < n; i++) for (int j = i + 1; j < n; j++) if (x[i] < x[j] == y[i] < y[j]) ans++;
        return ans;
    }
}

File: src/cses_fi/task2216/DummySolution.kt
package cses_fi.task2216

import java.util.*

object DummySolution {
    @JvmStatic
    fun numberOfRounds(nums: IntArray): Int {
        val n = nums.size
        if (n <= 1) return n;

        val pq = PriorityQueue(nums.asList())
        val used = BitSet(n)
        var ans = 0
        var i = 0
        while (!pq.isEmpty()) {
            if (i == 0) ans++
            if (!used[i] && pq.peek() == nums[i]) {
                used[i] = true
                pq.poll()
            }
            if (++i == n) i = 0
        }
        return ans
    }
}
